Source/common/SNTCommonEnums.h

```diff
@@ -24,6 +24,7 @@ typedef NS_ENUM(NSInteger, SNTRuleType) {
 
   SNTRuleTypeBinary = 1,
   SNTRuleTypeCertificate = 2,
+  SNTRuleTypePackage = 3,
```

Comment:
Since this new rule type is just used as an indication to show a notification we don't need this enum. The package rule will never be stored as a rule.